content/automate/ManageIQ/Transformation/TransformationHosts/Common.class/__methods__/killvirtv2v.rb

```diff
@@ -0,0 +1,45 @@
+module ManageIQ
+  module Automate
+    module Transformation
+      module TransformationHost
+        module Common
+          class VMCheckTransformed
+            def initialize(handle = $evm)
+              @debug = true
+              @handle = handle
+            end
+
+            def main
+              require 'json'
+
+              task = @handle.vmdb(:service_template_transformation_plan_task).find_by(:id => @handle.root['service_template_transformation_plan_task_id'])
+
+              # Retrieve transformation host
+              transformation_host = @handle.vmdb(:host).find_by(:id => task.get_option(:transformation_host_id))
+
+              # Skip if virtv2v has not started yet or is already finished
```

Comment:
@fdupont-redhat 
Can all of these skips be in a separate function called validate or valid and you call the remote command only if its valid.